lego-webapp/components/CollapsibleDisplayContent/index.tsx

```diff
@@ -57,15 +51,7 @@ function CollapsibleDisplayContent({
           className={className}
           ref={ref}
         >
-          <Editor
-            onChange={() => {}}
-            onBlur={() => {}}
-            onFocus={() => {}}
-            value={content}
-            placeholder={placeholder}
-            disabled
-            domParser={import.meta.env.SSR ? pageContext.domParser : undefined}
-          />
+          <EditorContent content={content ?? ''} />
```

Comment:
```suggestion
          <EditorContent content={content} />
```
I think `EditorContent` should handle if content is undefined so we don't need to do it every time we want to use it. This applies to more places than this.